include/silo/test/query_fixture.test.h

```diff
@@ -84,26 +84,28 @@ class QueryTestFixture : public ::testing::TestWithParam<QueryTestScenario> {
       auto now = std::chrono::system_clock::now();
       auto duration = now.time_since_epoch();
       auto millis = std::chrono::duration_cast<std::chrono::milliseconds>(duration).count();
+      std::filesystem::path input_directory = fmt::format("test{}", millis);
+      std::filesystem::create_directories(input_directory);
+
+      std::ofstream config_file(input_directory / "preprocessing_config.yaml");
+      assert(config_file.is_open());
+      config_file << "inputDirectory: " << input_directory << "\n";
+      config_file << "ndjsonInputFilename: input.ndjson\n";
+      config_file << "intermediateResultsDirectory: " << input_directory / "temp\n";
+      config_file.close();
```

Comment:
This would require a new abstraction layer, which I was hesitant to add

Reply:
I don't see why? Can't you simply create the preprocessing config object in C++ code?

Reply:
Then this would be the only usage of the accessors to the fields of the class, which I would need to add, or the fields would need to be public instead of private, as they are now

Reply:
.. or a new subclass of AbstractConfig needs to be added that
(i) reads the YAML from memory instead of from a file on disc
(ii) has members to overwrite single fields

Reply:
But I hardly doubt that we would need that functionality, which makes it kind of unnecessary to add it here. The file on disc will be deleted after the test anyways